.github/workflows/lint.yaml

```diff
@@ -13,10 +13,13 @@
 # limitations under the License.
 
 name: lint
-on: [pull_request]
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled]
 
 jobs:
   build:
+    if: ${{ github.event.action != 'labeled' || github.event.label.name == 'actions:run' }}
```

Comment:
Shouldn't these try to remove the label as well?

Reply:
Well depends if we want two different labels? Do we want one for `tests: run` and one for `actions:run`? I currently have one label that will trigger the lint, code coverage check , integration tests, and unit tests all at the same time when the single label is added. I currently have only one of the jobs removing the label, I guess I could add it to all of them and  instead of throwing an error if label doesn't exist i can just have it try to remove the label, and exit peacefully if it can't (i.e if one of the other jobs has already removed it).